academy/4-ibc/go-relayer.md

```diff
@@ -130,6 +139,24 @@ And check the config with:
 $ rly config show
 ```
 
+<HighlightBox type="info">
+
+By default, transactions will be relayed with a memo of `rly(VERSION)` e.g. `rly(v2.0.0)`.
+
+To customize the memo for all relaying, use the `--memo` flag when initializing the configuration.
+
+```shell
+$ rly config init --memo "My custom memo"
+```
+
+Custom memos will have `rly(VERSION)` appended. For example, a memo of `My custom memo` running on relayer version `v2.0.0` would result in a transaction memo of `My custom memo | rly(v2.0.0)`.
```

Comment:
Just a question, is `My custom memo | rly(v2.0.0)` the resulting string, so `|` acting as a separator for later decoding? Or are you indicating a concatenation with the `|`? Then I would prefer to go with a `+` or `&`.

Reply:
I've yet to try it out but pretty sure it would just be part of the resulting string, similar to what hermes has